### PR TITLE
feat(diagnostics):  Add a reload command and support command keybinds

### PR DESCRIPTION
**Pull request by ThomasOrs**
## Summary
- Adds a vscode command to reload in the command pallete `>Minecraft Diagnostics: Reload MC`
  - Added a keyboard shortcut for this `ctrl+shift+r` this can be edited by the user in keyboard shortcuts
- Added a vscode command which allows users to add custom keyboard shortcuts to run commands from `keybindings.json`
  - Example `{ "key": "ctrl+shift+t", "command": "minecraft-debugger.runMinecraftCommand", "args": "say hello" },`
 - Added an error message if you have an active debug session but try and run an empty command from within the home view

## To test:
- Launch a debug session of the extension
**Reload**
- Try and run the reload command without a session active an observe the warning message
- Start a debugging session
- Run the reload command from the command pallete, confirm it reloads in game
- Run the reload command from hot keys `ctrl+shift+r`, confirm it reloads in game
**Command Keybinds**
- Open the `keybindings.json` (stored in Roaming/Code/User/keybindings.json) or command pallete `>Preferences: Open Keyboard Shortcuts (JSON)`
- Add a keybinding such as:
  - ```[{ "key": "ctrl+shift+t", "command": "minecraft-debugger.runMinecraftCommand", "args": "say hello" }]```
- Save and use the keybind
- Confirm Scripting says hello

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,6 +56,16 @@
             {
                 "command": "minecraft-debugger.showMinecraftDiagnostics",
                 "title": "Minecraft Diagnostics: Show"
+            },
+            {
+                "command": "minecraft-debugger.minecraftReload",
+                "title": "Minecraft Diagnostics: Reload MC "
```
Comment (chmeyer-ms) [inline package.json:62]: Probably not a "diagnostics" command.  Maybe call it "Minecraft Reload"

File: package.json [hunk 2]
```diff
@@ -56,6 +56,16 @@
...
+            }
+        ],
+        "keybindings":[
+            {
+                "command": "minecraft-debugger.minecraftReload",
```
Comment (chmeyer-ms) [inline package.json:67]: Not sure if we should stomp the default key bind with ours, without conditions (assuming you can add conditions).  Maybe we don't add this automatically and we let the user define it.  @JakeShirley  thoughts?

File: src/extension.ts [hunk 3]
```diff
@@ -1,4 +1,3 @@
-
```
Comment (chmeyer-ms) [inline src/extension.ts:1]: It's weird but the leading blank line is intentional :)

**Review by chmeyer-ms — APPROVED** [review]
Looks good, thanks for doing this.

File: package.json (hunk 1 above)
Comment (ThomasOrs) [reply package.json]: Renamed!

File: package.json (hunk 2 above)
Comment (ThomasOrs) [reply package.json:67]: Narrowed down the keybinding condition to:
- When in debugMode **and**
- When in a JS / TS File

Comment (JakeShirley) [reply package.json:67]: @ThomasOrs mind linking to the strings that are allowed there?  What syntax is that?

Comment (ThomasOrs) [reply package.json:67]: @JakeShirley https://code.visualstudio.com/api/references/when-clause-contexts
Here you go!

**Pull request merged**